.github/workflows/gen.yaml

```diff
@@ -10,8 +10,11 @@ jobs:
     steps:
     - name: Checkout repository
       uses: actions/checkout@v3
+    - name: check CPU architecture
+      shell: bash
+      run: cat /proc/cpuinfo && uname -a
```

Comment:
Do we still need this?

Reply:
Sorry, I forgot to remove this.
removed [b06e221](https://github.com/pipe-cd/pipecd/pull/4970/commits/b06e22198f322bf8f866fdc045ad0094a6bc7f0f)